components/BrandedButton/EditButton.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <BrandedButton
+    color="danger"
```

Comment:
Danger seems a little bit overkill for this kind of action but why not :-)

Reply:
This is the current color on udata-front, but I agree that we could change it later